src/shared/port-util.js

```diff
@@ -35,7 +37,7 @@ export function isMessage(data) {
  * Return true if the data payload from a MessageEvent matches `message`.
  *
  * @param {any} data
- * @param {Message} message
+ * @param {Partial<Message>} message
```

Comment:
This allows `isMessageEqual` to match `data` against a subset of fields of a message. In particular, to ignore the `requestId`.